src/librbd/internal.cc

```diff
@@ -68,6 +68,11 @@ namespace librbd {
     return image_name + RBD_SUFFIX;
   }
 
+  const string object_map_name(const string &image_id)
+  {
+    return RBD_OBJECT_MAP_PREFIX + image_id;
```

Comment:
I'm wondering if we should add the snapid to the object name, so that it would be simple to add an objectmap to an existing snapshot if it was inconsistent when snapshotted, or we added the ability to enable objectmap for existing images